client/src/App.tsx

```diff
@@ -1,7 +1,9 @@
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 import GlobalStyle from "./GlobalStyle";
-import Login from "./pages/Login";
-import SignUp from "./pages/SignUp";
+import Login from "./pages/login";
```

Comment:
컴포넌트의 이름은 대문자로 시작하는 규칙이 있습니다.

```suggestion
import Login from "./pages/Login";
```

Reply:
감사합니다! 해당 내용 수정하겠습니다!